### DISCUSSION TITLE
[openimageio] Building OIIO Python bindings for specific Python version?

### DISCUSSION THREAD

**Discussion by joaovbs96** [microsoft/vcpkg#37026] [Q&A]
I wanted to use vcpkg to build the Python bindings of OIIO, but I need to support multiple Python versions. By default, (if I'm not mistaken) Python 3.11 is used, but is there any way to instead use a specific Python version or a Python installation of my system?

Ideally, I would want to build the bindings for multiple Python version (3.9 to 3.12, for example) instead of just one specific version.

Thanks in advance!

**Comment by Osyotr** [comment]
Sadly, there's no easy way to achieve this.
One option is to create overlay ports for each python version you need.
Another option is to use python's [limited api](https://docs.python.org/3/c-api/stable.html#limited-c-api).
